go-controller/pkg/ovn/address_set.go

```diff
@@ -48,12 +59,9 @@ func NewOvnAddressSetFactory() AddressSetFactory {
 	return &ovnAddressSetFactory{}
 }
 
-// ovnAddressSetFactory implements the AddressSetFactory interface
-var _ AddressSetFactory = &ovnAddressSetFactory{}
-
 // NewAddressSet returns a new address set object
 func (asf *ovnAddressSetFactory) NewAddressSet(name string, ips []net.IP) (AddressSet, error) {
```

Comment:
This should be named `NewAddressSets`?

Reply:
Thanks @abhat Decision was to keep the abstraction and the methods to refer to AddressSet . Hence I've kept the same name.